neurom/check/neuron_checks.py

```diff
@@ -304,6 +304,10 @@ def has_no_dangling_branch(neuron):
                                                          for n in iter_neurites(neuron)
                                                          if n.type != NeuriteType.axon)))
 
+    if len(dendritic_points) == 0:
+        # returns True if no dendrites are present
+        return CheckResult(True)
```

Comment:
The behavior is incorrect, a dangling axon would then go undetected. I'll have a look tomorrow.

Reply:
yes, I'm not sure about what dangling is. It seems to be w.r.t to dendrite's first points, but then if there are no dendrites, it cannot be dangling...

Reply:
I agree with @wizmer, I think the non-dendritic trees should be checked as well.